folium/plugins/draw.py

```diff
@@ -43,37 +54,48 @@ class Draw(MacroElement):
                 });
                drawnItems.addLayer(layer);
              });
-
-        {{this._parent.get_name()}}.on('draw:created', function(e) {
-            drawnItems.addLayer(e.layer);
-        });
-
-        document.getElementById('export').onclick = function(e) {
-            var data = drawnItems.toGeoJSON();
-            var convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(data));
-            document.getElementById('export').setAttribute('href', 'data:' + convertedData);
-            document.getElementById('export').setAttribute('download','data.geojson');
-        }
+            {{this._parent.get_name()}}.on('draw:created', function(e) {
+                drawnItems.addLayer(e.layer);
+            });
+            document.getElementById('export').onclick = function(e) {
+              var data = drawnItems.toGeoJSON();
+              var convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(data));
+              document.getElementById('export').setAttribute('href', 'data:' + convertedData);
+              document.getElementById('export').setAttribute(
+                'download',
+                "{{kwargs.get('filename', 'data.geojson')}}"
+              );
+            }
             {% endmacro %}
             """)
 
-    def __init__(self, export=False):
+    js_url = 'https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/1.0.2/leaflet.draw.js'
```

Comment:
Why did you move the URLs here? Throughout folium they are in the `render` method, so I'd prefer to keep them there for consistency.